src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -630,7 +630,17 @@ ReactDOMComponent.Mixin = {
       var ownerDocument = nativeContainerInfo._ownerDocument;
       var el;
       if (namespaceURI === DOMNamespaces.html) {
-        el = ownerDocument.createElement(this._currentElement.type);
+        if (this._tag === 'script') {
+          // Create the script via .innerHTML so its "parser-inserted" flag is
+          // set to true and it does not execute
+          var div = ownerDocument.createElement('div');
+          div.innerHTML =
+            '<' + this._currentElement.type +
+            '></' + this._currentElement.type + '>';
```

Comment:
I guess we know here that this is `"script"` so we probably could just hard code that string. Otherwise, template strings?